src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandBiasTest.java

```diff
@@ -73,14 +73,17 @@
 
             foundData = true;
             int[] data;
-            if ( g.getAnyAttribute(GATKVCFConstants.STRAND_BIAS_BY_SAMPLE_KEY).getClass().equals(String.class)) {
-                final String sbbsString = (String)g.getAnyAttribute(GATKVCFConstants.STRAND_BIAS_BY_SAMPLE_KEY);
+	    Object sbbsObj = g.getAnyAttribute(GATKVCFConstants.STRAND_BIAS_BY_SAMPLE_KEY);
+            if (sbbsObj instanceof String) {
+                final String sbbsString = (String)sbbsObj;
                 data = encodeSBBS(sbbsString);
-            } else if (g.getAnyAttribute(GATKVCFConstants.STRAND_BIAS_BY_SAMPLE_KEY).getClass().equals(ArrayList.class)) {
+            } else if (sbbsObj instanceof List) {
```

Comment:
`instanceof` can be much more expensive than `getClass()` -- we should check if there's any noticeable effect on performance here before merging.

Reply:
I would argue that instanceof is actually the correct check here and comparing class is a subtle bug that will cause confusing issues if we ever encounter a different list type...

Reply:
I disagree @lbergelson -- notice that there is an unconditional cast to `ArrayList` just below anyway. And `String` is a `final` class, so the `instanceof` check there has no advantage over `getClass()`.